.github/workflows/build_linux_jax_wheels.yml

```diff
@@ -0,0 +1,117 @@
+name: Build Linux PyTorch Wheels
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        required: true
+        type: string
+      python_versions:
+        required: true
+        type: string
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
+        required: true
+        type: string
+      cloudfront_url:
+        description: CloudFront URL pointing to Python index
+        required: true
+        type: string
+      rocm_version:
+        description: ROCm version to install
+        type: string
+      tar_url:
+        description: URL to TheRock tarball to build against
+        type: string
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        required: true
+        type: string
+      python_versions:
+        required: true
+        type: string
+        default:
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
+        type: string
+        default: "v2"
+      cloudfront_url:
+        description: CloudFront base URL pointing to Python index
+        type: string
+        default: "d25kgig7rdsyks.cloudfront.net"
+      rocm_version:
+        description: ROCm version to install (e.g. "==1.0", ">1.0")
+        type: string
+      tar_url:
+        description: URL to TheRock tarball to build against
+        type: string
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build_jax_wheels:
+    strategy:
+      matrix:
+        jax_ref: [master]
+    name: Build Linux JAX Wheels | ${{ inputs.amdgpu_family }} | Python ${{ inputs.python_version }}
+    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
+    env:
+      PACKAGE_DIST_DIR: ${{ github.workspace }}/jax_rocm_plugin/wheelhouse
+      S3_BUCKET_PY: "therock-${{ inputs.release_type }}-python"
+    steps:
+      - name: Checkout JAX
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: rocm/rocm-jax
+          ref: ${{ inputs.jax_ref }}
+
+      - name: Configure Git Identity
+        run: |
+          git config --global user.name "therockbot"
+          git config --global user.email "therockbot@amd.com"
+
+      - name: Build JAX Wheels
+        run: |
+          cd rocm-jax
+          python3 build/ci_build \
```

Comment:
It might make sense to move this script over to TheRock, similar to what we have for PyTorch. Wdyt?

Reply:
No. This is the script that all CI teams use to build JAX. TheRock aren't the only users. DevOps team uses this when they build stuff against regular ROCm, and we use this in our CI on the JAX team.

Reply:
It might make more sense then to checkout TheRock in `rocm-jax`, similar to what is done in `rocm-libraries`?

Reply:
You guys wanted the opposite above, so I changed it. I can change it back if you want?

When this workflow checks out TheRock, all it uses is `./build_tools/third_party/s3_management/manage.py`. The actual build uses the tarball that you give it.